src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -123,6 +133,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $maxLength = $this->configuration->get(PasswordPolicyConfiguration::CONFIGURATION_MAXIMUM_LENGTH);
         $minLength = $this->configuration->get(PasswordPolicyConfiguration::CONFIGURATION_MINIMUM_LENGTH);
 
+        if ($options['show_guest_field'] === true) {
+            $builder
+                ->add('is_guest', SwitchType::class, [
+                    'label' => $this->trans('Register as guest', 'Admin.Global'),
```

Comment:
```suggestion
                    'label' => $this->trans('Create guest account', 'Admin.Global'),
```

Reply:
Would also work, but I am thinking that in the future, we could probably also display this checkbox in the "edit" form, so maybe we should avoid the words `Register` and `Create`.

So what about `Guest customer` or `Guest account` or just `Guest`?

Reply:
IMHO it should be just `Guest account`

Reply:
Let's stick with `Guest account` then. It seems clear enough to me. I'll edit my suggestion